iOS-Email-Client/AsyncTasks/SendMailAsyncTask.swift

```diff
@@ -215,20 +215,20 @@ class SendMailAsyncTask {
         }
         APIManager.postMailRequest(requestParams, token: myAccount.jwt, queue: queue) { (error, data) in
             if let error = error {
-                self.handleResponseInMainThread {
+                DispatchQueue.main.async {
                     self.setEmailAsFailed()
                     completion(error, nil)
                 }
                 return
             }
             let key = self.updateEmailData(data)
-            self.handleResponseInMainThread {
+            DispatchQueue.main.async {
                 DBManager.refresh()
                 guard let myKey = key else {
```

Comment:
you could just pass `completion(nil, key)` there's nothing different being done if the key doesn't exist